docs/resources/synthetics.md

```diff
@@ -28,8 +28,12 @@ resource "datadog_synthetics_test" "test_api" {
       target = "200"
   }
   locations = [ "aws:eu-central-1" ]
-  options = {
+  options_list {
     tick_every = 900
+
+    retry = {
+      count = 2
+    }
```

Comment:
That should be `retry_count` right?

Reply:
since we're in a `retry` block I guess the `retry_` prefix isn't needed? In the api the object is directly `retry: {count: int, interval: int}`

Reply:
Yeah but you implemented it with `retry_count` and `retry_interval` in options? Unless your implementation strategy is different in `options_list`?

Reply:
I had to add the prefix in `options` since we couldn't use a nested map, but with `options_list` we can keep this simple

Reply:
Makes sense.